src/test/java/com/coravy/hudson/plugins/github/GithubLinkActionFactoryTest.java

```diff
@@ -0,0 +1,45 @@
+package com.coravy.hudson.plugins.github;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.empty;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import com.coravy.hudson.plugins.github.GithubLinkAction.GithubLinkActionFactory;
+
+import hudson.model.Action;
+
+public class GithubLinkActionFactoryTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
Also it bad practise to add types into variable names, i.e. `myCoolList = new ArrayList()`